runner/src/server/views/layout.html

```diff
@@ -191,6 +191,34 @@
             text: backLinkText
         }) }}
     {% endif %}
+    
+    {% if changeRequests.length > 0 %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-!-margin-top-4">
+                <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+                    <div class="govuk-notification-banner__header">
+                        <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                            Change request
+                        </h2>
+                    </div>
+                    <div class="govuk-notification-banner__content">
+                        {% for question in changeRequests %}
+                            <p class="govuk-body govuk-!-margin-bottom-0">
+                                <strong>{{ question.title }}</strong>
+                            </p>
+                            <ul>
```

Comment:
You'll end up with more or less the same result but for consistency this should probably be marked up as
```html
<ul class="govuk-list govuk-list--bullet">
    {% for message in question.messages %}
        <li>{{ message }}</li>
    {% endfor %}
</ul>
```

Reply:
Changed!